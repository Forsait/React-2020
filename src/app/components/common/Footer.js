// @flow
import React from 'react';

import Brand from './Brand';

const footerStyle = {
  justifyContent: 'center',
};

export default class Footer extends React.Component<void> {
  render() {
    return (
      <div style={footerStyle} className="westside">
        <Brand />
      </div>
    );
  }
}
