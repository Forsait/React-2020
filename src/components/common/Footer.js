import React from 'react';

import Brand from './Brand';

const footerStyle = {
  justifyContent: 'center',
};

export default class Footer extends React.Component {
  render() {
    return <div style={footerStyle} className={'westside'}>
      <Brand />
    </div>
  }
}