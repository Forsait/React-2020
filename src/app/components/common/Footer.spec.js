import React from 'react';
import { shallow } from 'enzyme';

import Footer from './Footer';

describe('Footer component', () => {

  it('should be render correctly', () => {
    const com = shallow(<Footer />);
    expect(com).toMatchSnapshot()
  });

});