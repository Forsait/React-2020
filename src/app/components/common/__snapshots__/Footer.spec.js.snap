// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should be render correctly 1`] = `
<div
  className="westside"
  style={
    Object {
      "justifyContent": "center",
    }
  }
>
  <Brand />
</div>
`;
